diff --git a/node_modules/@react-native-async-storage/async-storage/android/build.gradle b/node_modules/@react-native-async-storage/async-storage/android/build.gradle
--- a/node_modules/@react-native-async-storage/async-storage/android/build.gradle
+++ b/node_modules/@react-native-async-storage/async-storage/android/build.gradle
@@ -39,33 +39,33 @@
 android {
     def agpVersion = com.android.Version.ANDROID_GRADLE_PLUGIN_VERSION
     if (agpVersion.tokenize('.')[0].toInteger() >= 7) {
-        namespace "com.reactnativecommunity.asyncstorage"
+        namespace = "com.reactnativecommunity.asyncstorage"
         buildFeatures {
-            buildConfig true
+            buildConfig = true
         }
     }
 
-    compileSdkVersion project.ext.AsyncStorageConfig.compileSdkVersion
+    compileSdkVersion = project.ext.AsyncStorageConfig.compileSdkVersion
     // Used to override the NDK path/version by allowing users to customize
     // the NDK path/version from their root project (e.g. for M1 support)
     if (rootProject.hasProperty("ndkPath")) {
-        ndkPath rootProject.ext.ndkPath
+        ndkPath = rootProject.ext.ndkPath
     }
     if (rootProject.hasProperty("ndkVersion")) {
-        ndkVersion rootProject.ext.ndkVersion
+        ndkVersion = rootProject.ext.ndkVersion
     }
 
 
     defaultConfig {
-        minSdkVersion project.ext.AsyncStorageConfig.minSdkVersion
-        targetSdkVersion project.ext.AsyncStorageConfig.targetSdkVersion
+        minSdkVersion = project.ext.AsyncStorageConfig.minSdkVersion
+        targetSdkVersion = project.ext.AsyncStorageConfig.targetSdkVersion
         buildConfigField "Long", "AsyncStorage_db_size", "${project.ext.AsyncStorageConfig.databaseSizeMB}L"
         buildConfigField "boolean", "AsyncStorage_useDedicatedExecutor", "${project.ext.AsyncStorageConfig.useDedicatedExecutor}"
         buildConfigField "boolean", "AsyncStorage_useNextStorage", "${useNextStorage}"
         buildConfigField "boolean", "IS_NEW_ARCHITECTURE_ENABLED", "${isNewArchitectureEnabled}"
     }
     lintOptions {
-        abortOnError false
+        abortOnError = false
     }
 
     if (useNextStorage) {
@@ -95,7 +95,7 @@
 repositories {
     maven {
         // All of React Native (JS, Obj-C sources, Android binaries) is installed from npm
-        url "${project.ext.resolveModulePath("react-native")}/android"
+        url = "${project.ext.resolveModulePath("react-native")}/android"
     }
     google()
     mavenCentral()
