diff --git a/node_modules/@react-native-clipboard/clipboard/android/build.gradle b/node_modules/@react-native-clipboard/clipboard/android/build.gradle
--- a/node_modules/@react-native-clipboard/clipboard/android/build.gradle
+++ b/node_modules/@react-native-clipboard/clipboard/android/build.gradle
@@ -63,23 +63,23 @@
 android {
 	def agpVersion = com.android.Version.ANDROID_GRADLE_PLUGIN_VERSION
 	if (agpVersion.tokenize('.')[0].toInteger() >= 7) {
-	  namespace "com.reactnativecommunity.clipboard"
+	  namespace = "com.reactnativecommunity.clipboard"
 	}
 	
-    compileSdkVersion getExtOrIntegerDefault('compileSdkVersion')
+    compileSdkVersion = getExtOrIntegerDefault('compileSdkVersion')
 
     // Used to override the NDK path/version on internal CI or by allowing
     // users to customize the NDK path/version from their root project (e.g. for M1 support)
     if (rootProject.hasProperty("ndkPath")) {
-        ndkPath rootProject.ext.ndkPath
+        ndkPath = rootProject.ext.ndkPath
     }
     if (rootProject.hasProperty("ndkVersion")) {
-        ndkVersion rootProject.ext.ndkVersion
+        ndkVersion = rootProject.ext.ndkVersion
     }
 
     defaultConfig {
-        minSdkVersion getExtOrIntegerDefault('minSdkVersion')
-        targetSdkVersion getExtOrIntegerDefault('targetSdkVersion')
+        minSdkVersion = getExtOrIntegerDefault('minSdkVersion')
+        targetSdkVersion = getExtOrIntegerDefault('targetSdkVersion')
     }
 
     sourceSets.main {
