diff --git a/node_modules/expo/android/build.gradle b/node_modules/expo/android/build.gradle
--- a/node_modules/expo/android/build.gradle
+++ b/node_modules/expo/android/build.gradle
@@ -36,14 +36,14 @@
 
 expoModule {
   // We can't prebuild the module because it depends on the generated files.
-  canBePublished false
+  canBePublished = false
 }
 
 android {
-  namespace "expo.core"
+  namespace = "expo.core"
   defaultConfig {
-    versionCode 1
-    versionName "54.0.30"
+    versionCode = 1
+    versionName = "54.0.30"
     consumerProguardFiles("proguard-rules.pro")
   }
   testOptions {
