diff --git a/node_modules/expo-modules-core/android/build.gradle b/node_modules/expo-modules-core/android/build.gradle
--- a/node_modules/expo-modules-core/android/build.gradle
+++ b/node_modules/expo-modules-core/android/build.gradle
@@ -64,22 +64,22 @@
 def shouldTurnWarningsIntoErrors = findProperty("EXPO_TURN_WARNINGS_INTO_ERRORS") == "true"
 
 expoModule {
-  canBePublished false
+  canBePublished = false
 }
 
 android {
   if (rootProject.hasProperty("ndkPath")) {
-    ndkPath rootProject.ext.ndkPath
+    ndkPath = rootProject.ext.ndkPath
   }
   if (rootProject.hasProperty("ndkVersion")) {
-    ndkVersion rootProject.ext.ndkVersion
-  }
-
-  namespace "expo.modules"
+    ndkVersion = rootProject.ext.ndkVersion
+  }
+
+  namespace = "expo.modules"
   defaultConfig {
     consumerProguardFiles 'proguard-rules.pro'
-    versionCode 1
-    versionName "3.0.29"
+    versionCode = 1
+    versionName = "3.0.29"
     buildConfigField "String", "EXPO_MODULES_CORE_VERSION", "\"${versionName}\""
     buildConfigField "boolean", "IS_NEW_ARCHITECTURE_ENABLED", isNewArchitectureEnabled.toString()
 
@@ -106,9 +106,9 @@
   }
 
   buildFeatures {
-    buildConfig true
-    prefab true
-    compose shouldIncludeCompose
+    buildConfig = true
+    prefab = true
+    compose = shouldIncludeCompose
   }
 
   packagingOptions {
