diff --git a/node_modules/expo-constants/android/build.gradle b/node_modules/expo-constants/android/build.gradle
--- a/node_modules/expo-constants/android/build.gradle
+++ b/node_modules/expo-constants/android/build.gradle
@@ -10,14 +10,14 @@
 
 expoModule {
   // We can't prebuild the module because we need to apply `get-app-config-android.gradle` script.
-  canBePublished false
+  canBePublished = false
 }
 
 android {
-  namespace "expo.modules.constants"
+  namespace = "expo.modules.constants"
   defaultConfig {
-    versionCode 33
-    versionName "18.0.12"
+    versionCode = 33
+    versionName = "18.0.12"
   }
 }
 
