diff --git a/node_modules/react-native-reanimated/android/build.gradle b/node_modules/react-native-reanimated/android/build.gradle
--- a/node_modules/react-native-reanimated/android/build.gradle
+++ b/node_modules/react-native-reanimated/android/build.gradle
@@ -127,7 +127,7 @@
 def REANIMATED_FEATURE_FLAGS = getReanimatedStaticFeatureFlags()
 
 // Set version for prefab
-version REANIMATED_VERSION
+version = REANIMATED_VERSION
 
 def reanimatedPrefabHeadersDir = project.file("$buildDir/prefab-headers/reanimated")
 
@@ -157,34 +157,34 @@
 apply plugin: "de.undercouch.download"
 
 android {
-    compileSdkVersion safeExtGet("compileSdkVersion", 34)
-
-    namespace "com.swmansion.reanimated"
+    compileSdkVersion = safeExtGet("compileSdkVersion", 34)
+
+    namespace = "com.swmansion.reanimated"
 
     if (rootProject.hasProperty("ndkPath")) {
-        ndkPath rootProject.ext.ndkPath
+        ndkPath = rootProject.ext.ndkPath
     }
     if (rootProject.hasProperty("ndkVersion")) {
-        ndkVersion rootProject.ext.ndkVersion
+        ndkVersion = rootProject.ext.ndkVersion
     }
 
     buildFeatures {
-        prefab true
-        prefabPublishing true
-        buildConfig true
+        prefab = true
+        prefabPublishing = true
+        buildConfig = true
     }
 
     prefab {
         reanimated {
-            headers reanimatedPrefabHeadersDir.absolutePath
+            headers = reanimatedPrefabHeadersDir.absolutePath
         }
     }
 
     defaultConfig {
-        minSdkVersion safeExtGet("minSdkVersion", 23)
-        targetSdkVersion safeExtGet("targetSdkVersion", 34)
-        versionCode 1
-        versionName REANIMATED_VERSION
+        minSdkVersion = safeExtGet("minSdkVersion", 23)
+        targetSdkVersion = safeExtGet("targetSdkVersion", 34)
+        versionCode = 1
+        versionName = REANIMATED_VERSION
 
         buildConfigField("boolean", "REANIMATED_PROFILING", REANIMATED_PROFILING.toString())
         buildConfigField("String", "REANIMATED_VERSION_JAVA", "\"${REANIMATED_VERSION}\"")
@@ -225,7 +225,7 @@
         }
     }
     lintOptions {
-        abortOnError false
+        abortOnError = false
     }
     packagingOptions {
         // For some reason gradle only complains about the duplicated version of librrc_root and libreact_render libraries
