diff --git a/node_modules/react-native-udp/android/build.gradle b/node_modules/react-native-udp/android/build.gradle
--- a/node_modules/react-native-udp/android/build.gradle
+++ b/node_modules/react-native-udp/android/build.gradle
@@ -21,7 +21,7 @@
 }
 
 android {
-    compileSdkVersion safeExtGet('compileSdkVersion', 31)
+    compileSdkVersion = safeExtGet('compileSdkVersion', 31)
 
     compileOptions {
         sourceCompatibility JavaVersion.VERSION_1_8
@@ -29,18 +29,18 @@
     }
 
     defaultConfig {
-        minSdkVersion safeExtGet('minSdkVersion', 16)
-        targetSdkVersion safeExtGet('targetSdkVersion', 31)
+        minSdkVersion = safeExtGet('minSdkVersion', 16)
+        targetSdkVersion = safeExtGet('targetSdkVersion', 31)
     }
     lintOptions{
-        abortOnError false
+        abortOnError = false
     }
 }
 
 repositories {
     maven {
         // All of React Native (JS, Obj-C sources, Android binaries) is installed from npm
-        url "$rootDir/../node_modules/react-native/android"
+        url = "$rootDir/../node_modules/react-native/android"
     }
     google()
     mavenLocal()
